src/QsCompiler/Tests.Compiler/TestCases/CapabilityVerification.qs

```diff
@@ -191,4 +197,38 @@ namespace Microsoft.Quantum.Testing.CapabilityVerification {
     function ResultArray(rs : Result[]) : Bool {
         return rs == [One] ? true | false;
     }
+    
+    // Inferred capabilities can be overridden.
+
+    @Capability("Unknown")
+    operation ResetOverrideHigh(q : Qubit) : Unit {
+        if (M(q) == One) {
+            X(q);
+        }
+    }
+
+    @Capability("QPRGen0")
+    operation ResetOverrideLow(q : Qubit) : Unit {
+        if (M(q) == One) {
+            X(q);
+        }
+    }
+}
```

Comment:
It would be good to have another test case in which capability is overridden with `QPRGen1`. Maybe even two cases, one in which the if-statement contains a return statement and one in which the if-statement contains a set-statement for a mutable variable declared outside that block.

Reply:
I added tests for overriding QPRGen0 to QPRGen1, and for explicitly specifiying QPRGen1 on something that would already be inferred as QPRGen1.

> Maybe even two cases, one in which the if-statement contains a return statement and one in which the if-statement contains a set-statement for a mutable variable declared outside that block.

IMO it's not necessary to distinguish between these two cases in the override tests - there are other tests that make sure both of those cases are inferred as Unknown.

Reply:
Also just added a test for overriding Unknown to Gen1 (forgot about that case).

Reply:
Thanks Sarah! I liked the new names of the operations :)